PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,11 @@
 - remove
 - modify
 
-## QA steps (optional)
+## Reason for Change
```

Comment:
Will this eliminate the need to create an issue for every PR?

Reply:
no, you should have an issue, but it might be that you have 10 PRs for one issue for instance and the reasoning for specific PR changes isnt stated in the issue. Or alternatively, the issue might only state what is required, like delivering a feature, but you might in the PR need to do a lot of (related) work to get that feature completed.